.github/workflows/pr_build.yml

```diff
@@ -114,18 +128,20 @@ jobs:
         else
           echo "ARTIFACT_PATTERN=*.$INSTALLER_EXT" >> $GITHUB_ENV
         fi
+
     - name: Clear Working Directory
-      if: startsWith(matrix.os, 'windows')
+      if: startsWith(matrix.os, 'windows') || matrix.os == 'self-hosted_debian-11_aarch64'
```

Comment:
Does it make sense to make such a specific check here?

Reply:
I should check for only "self-hosted" instead, since this wouldn't harm a running 95% of the time.